### DISCUSSION TITLE
Promote a comment to new topic?

### DISCUSSION THREAD

**Discussion by bdarcus** [community/community#3789] [Discussions]
In the same way I can move a discussion to a new issue, I'd like to be able to move a comment within an existing topic to a new topic (for good, but off-topic, replies).

Possible to add that?

Sorry if this has already been raised, but quick review didn't turn anything up.

**Comment by theophanemayaud** [comment]
Yeah, I agree that'd be great, when some users use comments to actually ask for something new for example, or just start on a different topic !
